go/arrow/cdata/cdata.go

```diff
@@ -197,7 +197,7 @@ func importSchema(schema *CArrowSchema) (ret arrow.Field, err error) {
 
 	// handle types with params via colon
 	typs := strings.Split(f, ":")
-	defaulttz := "UTC"
+	defaulttz := ""
```

Comment:
was this incorrect by the spec? I thought I pulled this from the c bridge which had the same default? (I could be misremembering though)

Reply:
Yes, it was incorrect and failing the integration tests between C++ and Go.
A timestamp with an missing timezone is not the same as a timestamp with the UTC timezone, as explained in the format spec:
https://github.com/apache/arrow/blob/772a01c080ad57eb11e9323f5347472b769d45de/format/Schema.fbs#L264

Reply:
Bah, must have mistaken that when I originally coded this :frowning: